sprint8/project/style.css

```diff
@@ -0,0 +1,35 @@
+body {
+  background-color: rgba(237, 201, 130, 0.85);
+}
+
+.foodProducts {
+  display: flex;
+  align-items: flex-end;  /* ten kod nie działa, zeby divy były na dole stronie*/
+}
+
+.foodProducts__shopping {
+  font-size: 2,5rem;
+  width: 20%;
+  text-align: center;
+  font-family: sans-serif;
+}
+
+.foodProducts__shopping--yellow {
+  background-color: rgb(244, 211, 93);
+}
+
+.foodProducts__shopping--red {
+  background-color: rgb(219, 91, 74);
+}
+
+.foodProducts__shopping--pink {
+  background-color: rgb(246, 123, 125);
+}
+
+.foodProducts__shopping--beige {
+  background-color: rgb(226, 187, 127);
+}
+
+.foodProducts__shopping--brown {
+  background-color: rgb(161, 119, 90);
+}
```

Comment:
Kod wygląda o niebo lepiej :)
Narazie nie robimy divów na dole. Naprawimy to później. Na teraz zostawiamy na dole. Ale musi być to zgodne z projektem. Prosze się przyjrzeć projektowi. Robimy tak samo jak w projekcie oprócz tego żeby było na dole.
Proszę też poprawić szerokości divów tak jak pisałem w którymś komentarzu (szerokość obliczana automatycznie)